<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WsdlFileView</class>
 <widget class="QWidget" name="WsdlFileView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>327</width>
    <height>138</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WsdlFileView</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="WsdlFileNameLabel">
       <property name="text">
        <string/>
       </property>
       <property name="indent">
        <number>4</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="BrowseBtn">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="TableView">
     <property name="toolTip">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../PoamParser/PoamParserEyeRetina/PoamParserWriter/PoamParserWriter.qrc"/>
 </resources>
 <connections/>
</ui>
